### PR TITLE
Add a list of apps and integrations used on the project's Slack

### PR DESCRIPTION
**Pull request by sam-pires**
- Created the apps-and-integrations.md file
- Added Productboard to the page

### CODE REVIEW

File: slack/apps-and-integrations/apps-and-integrations.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# List of apps and integrations used
+
+## Productboard
+- **Description:** Productboard is a product management tool, used by the PrestaShop Core's product team to gather feedback, organize and pritoritize features and improvements to build, and to follow their development.
+The Productboard app connects this Slack to the PrestaShop's Productboard.  
+It allows the product managers to transform a Slack message (and its comments) into a feedback "note" on productbaord.  
+More info on the productboard website: https://www.productboard.com/
+- **Access:** Limited
```
Comment (ttoine) [inline slack/apps-and-integrations/apps-and-integrations.md:8]: can you please explain that there is also a public view and add the link?

File: slack/apps-and-integrations/apps-and-integrations.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+- **Users:** Only the PrestaShop's productboard users can create note from Slack
+- **Owners:** PrestaShop Core's product management team, coreproduct@prestashop.com
```
Comment (ttoine) [inline slack/apps-and-integrations/apps-and-integrations.md:10]:
```suggestion
- **Owners:** PrestaShop Core's product management team, coreproduct@prestashop.com, currently composed by PrestaShop employees only.
```

File: slack/apps-and-integrations/apps-and-integrations.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- **Language:** EN 
+- **Channels perimeter:** all channels
```
Comment (ttoine) [inline slack/apps-and-integrations/apps-and-integrations.md:12]: all public channels ? or all channels where one of your team has an access ?

Comment (sam-pires) [reply slack/apps-and-integrations/apps-and-integrations.md:12]: Well, all channels, but as I restricted the use to only PB users (on PB side) only a PM can collect a feedback. Thus, a PM needs to be in the channel to collect the feedback, but the app can be used in any channel (private and public)

File: slack/apps-and-integrations/apps-and-integrations.md (hunk 1 above)
Comment (sam-pires) [reply slack/apps-and-integrations/apps-and-integrations.md]: added the link to the PB Portal (http://feedback.prestashop.com/) though, people won't see their feedback there, they will only see the ideas that result from their feedback after we grouped them and organize them

**Pull request merged**